tensorflow_addons/optimizers/moving_average.py

```diff
@@ -82,6 +82,14 @@ def __init__(
         super().__init__(optimizer, sequential_update, name, **kwargs)
         self._num_updates = num_updates
         if self._num_updates is not None:
+            if isinstance(self._num_updates, tf.Variable):
+                tf.debugging.assert_integer(
+                    self._num_updates,
+                    (
+                        'type of argument "num_updates" must be '
+                        + f"int; got {self._num_updates.dtype} instead"
```

Comment:
We are compatible with Python3.5, so fstring should be avoided.

Reply:
Thanks. Updated!